src/domain/registration/utils.ts

```diff
@@ -181,6 +181,65 @@ export const copyRegistrationToSessionStorage = async (
 export const getRegistrationItemId = (id: string): string =>
   `registration-item-${id}`;
 
+export const formatSingleInstructions = ({
+  instructions,
+  lang,
+}: {
+  instructions: string;
+  lang: string;
+}): string => {
+  const dataMapping = {
+    fi: {
+      text1: 'Ilmoittautumisen teknisissä ongelmissa ole yhteydessä:',
+      text2: 'Palvelukeskus Helsinki',
+      text3: '09 310 25280, palveluaika klo 8-18',
+      expectedText: 'Palvelukeskus Helsinki',
```

Comment:
The same value is repeated for all languages and for both `text2` and `expectedText`. Could there be a single constant, for example `serviceCenterName`, that could be used in `trimmedText` / the output of this function?

Reply:
Fixed as suggested